docs: add open source commitment document (#1129)
ory/docs: docs/open-source/commitment.mdx

---
title: Ory is committed to Open Source
id: commitment
sidebar_title: Commitment
---

Ory is fundamentally rooted in open source, its community dynamics, and development principles. Ory's software is licensed under
the Apache 2.0 license. Ory's founders have decades of experience in open source and open knowledge.

To build open source software and help its community and adoption grow, Ory offers a commercial product which uses open source
software from Ory and others like the Linux foundation.

> ## Open Source is a good idea

Reviewer: ```suggestion
## Why Ory is developing Open Source```

Building a profitable open source business is non-trivial. But there are many good reasons why
[open source is the right choice](https://opensource.google/documentation/reference/why) for companies like Ory:

> - Engineering Economics: Releasing work as open source and the corresponding contribution process eventually result in a higher

Reviewer 2: Ah :D 

Hm, I read it as one item, as one comes from the other. It's a weird sentence... I'm fine with it either way, just wanted to catch a typo. :)

> - Engineering Economics: Releasing work as open source and the corresponding contribution process eventually result in a higher

Reviewer 3: ```suggestion
- Engineering economics: Releasing work as open source and the corresponding contribution process eventually result in a higher
```

Please follow "normal" [capitalization](https://en.wikipedia.org/wiki/Capitalization_in_English) rules 🙏

  return on the initial investment made versus the alternative closed source process.
- Establishing or Supporting an Open Standard: Publishing your project under an open source license can encourage adoption of a
  standard. When a project becomes a standard, you benefit from a massive influx of outside contributors so the project, and the
  ecosystem around it, evolves more rapidly.

> - Disintermediation: Releasing open source software can serve to decouple things and create an opening to compete in a new field.

Reviewer: ```suggestion
- Disintermediation: Releasing open source software can serve to decouple software components and create an opening to compete in a new field.
```

  Consider Kubernetes, software for automating management, deployment, operations and scaling of containerized applications.

> - Drive Adoption with a Platform and Ecosystem: Releasing a project as open source allows others to adapt and build on top of your

Reviewer: ```suggestion
- Driving Adoption of a Platform and Ecosystem: Releasing a project as open source allows others to adapt and build on top of your
```
